cfgov/jinja2/v1/_includes/organisms/expandable-group.html

```diff
@@ -23,33 +23,6 @@
 
    ========================================================================== #}
 
-{% if not value %}
-    {% do global_dict.update(
-        {
-            'expandable_group': {
-                'heading': 'Group Header',
-                'has_rule': true,
-                'expandables': [
-                    {
-                        'label': 'Expand 1 - initially open',
-                        'paragraph': lipsum(1, false, 10, 225),
-                        'is_bordered': true,
-                        'is_midtone': true,
-                        'is_expanded': true
-                    },
-                     {
-                        'label': 'Expand 2',
-                        'paragraph': lipsum(1, false, 10, 225),
-                        'is_bordered': true,
-                        'is_midtone': true,
-                    }
-                ],
-            }
-        })
-    %}
-{% endif %}
-{% set value = value or global_dict.expandable_group %}
-
 <div class="o-expandable-group
```

Comment:
same as my above expandables comment we need a conditional here for value or it breaks when being imported on the page template level. you can test this scenario by creating a `Browse Page` and trying to view the draft.
```
{% if value %}
```